migrate/v0_11/migrate.go

```diff
@@ -57,11 +57,92 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
 		}
 		swaps = append(swaps, newSwap)
 	}
+
+	// -------------- ADD BTCB To BEP3 params --------------------
+	btcbAssetParam := v0_11bep3.NewAssetParam(
+		"btcb",
+		0,
+		v0_11bep3.SupplyLimit{
+			Limit:          sdk.NewInt(10000000000), // 100 BTC limit at launch
+			TimeLimited:    false,
+			TimePeriod:     time.Duration(0),
+			TimeBasedLimit: sdk.ZeroInt()},
+		true,
+		mustAccAddressFromBech32("kava14qsmvzprqvhwmgql9fr0u3zv9n2qla8zhnm5pc"),
+		v0_9Params.BnbDeputyFixedFee,
+		sdk.OneInt(),
```

Comment:
The initial BEP3 module’s minimum fees should match the deputy min fees, which we've agreed will vary for different asset types. Here the minimum is set to `sdk.OneInt()` for all of the assets which will allow users to create swaps that the deputy may not relay.

**Scenario**: Deputy minimum fee for btcb is set to 10. The BEP3 module accepts a swap of 9 btcb but the Deputy will never relay it. The minimum swap amount for an asset should be equal to (or ideally greater than) the fixed deputy fee for that asset.

This is also an issue [here](https://github.com/Kava-Labs/kava/blob/master/migrate/v0_11/migrate.go#L25). For existing BEP3 assets (just bnb) lets set the new MinSwapAmount to the same value as the existing MinSwapAmount (1000 bnb).

Reply:
Good catch. I updated the fees for each asset to be ~$0.0025 and set the min swap amount to be one greater than the fee. Also of note, XRP is 8 decimals on binance chain, so updated to reflect that as well.

Reply:
Oh, this answers my question over in https://github.com/Kava-Labs/kava/pull/663 about XRP conversion factor.